eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -0,0 +1,48 @@
+function Get-javascript-OnboardedDocsMsPackages($DocRepoLocation) {
+  $packageOnboardingFiles = @(
+    "$DocRepoLocation/ci-configs/packages-latest.json",
+    "$DocRepoLocation/ci-configs/packages-preview.json",
+    "$DocRepoLocation/ci-configs/packages-legacy.json")
+
+  $onboardedPackages = @{}
+  foreach ($file in $packageOnboardingFiles) {
+    $onboardingSpec = ConvertFrom-Json (Get-Content $file -Raw)
+    foreach ($spec in $onboardingSpec.npm_package_sources) {
+      $packageName = $spec.name
+      if ($packageName.LastIndexOf('@') -gt 0) {
+        # Package has an '@' symbol deliminting the end of the package name
+        $packageName = $packageName.Substring(0, $packageName.LastIndexOf('@'))
+      }
+      $onboardedPackages[$packageName] = $null
+    }
+  }
+
+  return $onboardedPackages
+}
+
+function Get-javascript-DocsMsTocData($packageMetadata, $docRepoLocation) {
+  $packageLevelReadmeName = $packageMetadata.Package.Replace('@azure/', '').Replace('@azure-tools/', '').Replace('azure-', '');
+
+  if ($packageMetadata.Package.StartsWith('@azure-rest/')) {
+    $packageLevelReadmeName = "$($packageMetadata.Package.Replace('@azure-rest/', ''))-rest"
+
+    # TODO: Consider using metadata in doc repo /metadata folder to get
```

Comment:
Doesn't that already exist in the package metadata from the repo?

Reply:
We can get this information for packages where metadata is present in the `metadata` folder similar to how we do it in `Update-DocsMsPackages.ps1`.